onnx/version_converter/adapters/softmax_12_13.h

```diff
@@ -44,38 +54,19 @@ class Softmax_12_13 final : public Adapter {
       node->output()->setUniqueName(original_output_name + "_intermediate");
```

Comment:
Thanks for the PR. I wonder why we have the old lines 35-39? Specifically, `flatten` produces a 2D output, and so does it matter if the axis is 1 or -1 ? Don't they mean the same for a 2D tensor?

Just wondering, I realize it has nothing to do with the scope of the current PR.

Reply:
I think axis 1 and -1 mean the same. Let me simplify these lines in the next commit